optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/api/PlannerBenchmarkFactory.java

```diff
@@ -51,8 +51,9 @@
      * The XML root element must be {@code <solver>}.
      * <p>
      * To read an XML benchmark configuration instead, use {@link #createFromXmlResource(String)}.
+     * 
```

Comment:
I highly recommend to eliminate the trailing space. Trailing spaces are a style-smell, I would say. Git highlights it as an error when displaying a diff. IDEs can be configured to automatically strip off trailing white space on save.

I know this is a result of applying the maven plugin and I'd argue it's a bug. But note that it won't reintroduce the trailing white space if the Javadoc is already formatted correctly and contains the blank line after description.

Reply:
Concrete example why neglecting the white space produced by the plugin will be a problem:

I have my IDE configured to strip trailing spaces on save and I use my IDE to format the code using a shortcut. So each time I'll edit and commit a source file, I'll produce a formatting noise by removing the trailing space (until it's eventually eradicated).

I'd find that quite annoying especially given that such situation can be easily avoided. Remember that the maven plugin is a safety net for contributors using exotic IDEs or unwilling to import our style.

Reply:
Thanks for pointing this out, already fixed.

Reply:
Radovan removed all trailing spaces in all these javadocs